src/compact_filter.h

```diff
@@ -71,6 +71,26 @@ class SubKeyFilterFactory : public rocksdb::CompactionFilterFactory {
   Engine::Storage *stor_ = nullptr;
 };
 
+class TransitFilter : public rocksdb::CompactionFilter {
+ public:
+  const char *Name() const override { return "TransitFilter"; }
+  bool Filter(int level, const Slice &key, const Slice &value,
+              std::string *new_value, bool *modified) const override {
+    // only filter the propagating command here
```

Comment:
```suggestion
    // We propagate Lua commands which don't store data,
    // just in order to implement updating Lua state.
```
How about above comments